src/ml/federated/JasmineGraphFederatedInstance.cpp

```diff
@@ -0,0 +1,483 @@
+#include <stdio.h>
```

Comment:
Add license header

Reply:
File content merged to JasmineGraphServer in ffb1fdb

Reply:
Ok